playground/assets/__tests__/assets.spec.ts

```diff
@@ -276,6 +276,12 @@ describe('css url() references', () => {
     expect(bg).toMatch(assetMatch)
   })
 
+  test('preinlined SVG', async () => {
+    expect(await getBg('.css-url-preinlined-svg')).toMatch(
+      /data:image\/svg\+xml,.+/,
+    )
+  })
+
```

Comment:
When lightningcss is not used, preinlined SVGs were working  without this PR. I added this so that we don't break this usecase in future.